src/backend/db/database.py

```diff
@@ -16,12 +16,12 @@
 #     along with FMTM.  If not, see <https:#www.gnu.org/licenses/>.
 #
 
-from sqlalchemy import create_engine, MetaData
+from sqlalchemy import MetaData, create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
 # TODO don't hardcode this
-SQLALCHEMY_DATABASE_URL = 'postgresql+psycopg2://fmtm:fmtm@db:5432'
+SQLALCHEMY_DATABASE_URL = "postgresql+psycopg2://admin:admin@localhost:5432/postgres"
```

Comment:
Is this meant to be included?

Reply:
No , should be read from .env , I will include its url

Reply:
So you'll push the PR again after that fix to read the .env file ?

Reply:
yes , converting pr to draft to work on it